SignInCheckIn/SignInCheckIn/Controllers/ChildSigninController.cs

```diff
@@ -1,33 +1,30 @@
+//using Crossroads.ApiVersioning;
+using Crossroads.Web.Auth.Controllers;
+using Crossroads.Web.Common.Security;
+using Crossroads.Web.Common.Services;
+using MinistryPlatform.Translation.Repositories.Interfaces;
+using SignInCheckIn.Exceptions.Models;
+using SignInCheckIn.Models.DTO;
+using SignInCheckIn.Services.Interfaces;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Web.Http;
 using System.Web.Http.Description;
-using AutoMapper;
-using MinistryPlatform.Translation.Repositories.Interfaces;
-using SignInCheckIn.Exceptions.Models;
-using SignInCheckIn.Models.DTO;
-using SignInCheckIn.Security;
-using SignInCheckIn.Services.Interfaces;
-//using Crossroads.ApiVersioning;
-using Crossroads.Utilities.Services.Interfaces;
-using Crossroads.Web.Common.Security;
-using MinistryPlatform.Translation.Models.DTO;
-using Newtonsoft.Json.Linq;
-using Crossroads.Web.Common.Services;
 
 namespace SignInCheckIn.Controllers
 {
     [RoutePrefix("api")]
-    public class ChildSigninController : MpAuth
+    public class ChildSigninController : AuthBaseController
     {
         private readonly IWebsocketService _websocketService;
         private readonly IChildSigninService _childSigninService;
         private readonly IKioskRepository _kioskRepository;
         private readonly IContactRepository _contactRepository;
         private readonly IFamilyService _familyService;
+        private readonly int _kidsClubTools = 112;
```

Comment:
Constants should be pascal cased - so private const int KidsClubToolsRole = 112;